deployment/clowdapp.yml

```diff
@@ -61,8 +61,17 @@ objects:
             value: ${LOGLEVEL}
           - name: DB_IMPL
             value: ${DB_IMPL}
-          - name: DEBUG
-            value: ${DEBUG}
+
+          - name: GITHUB_WEBHOOK_SECRET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: github-secret-token
+                name: github-secret-token
+          - name: GITLAB_WEBHOOK_SECRET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: gitlab-secret-token
+                name: gitlab-secret-token
```

Comment:
Remember the way the secrets are named here, needs to match how they are named in vault.